metabrainz/templates/index/social-contract.html

```diff
@@ -0,0 +1,60 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ _('Social Contract') }} - MetaBrainz Foundation{% endblock %}
+
+{% block content %}
+  <h1 class="page-title">{{ _('Social Contract') }}</h1>
+
+  <p>
+    This social contract defines the spirit underpinning MetaBrainz' projects and its community. This contract is not a legally binding 
+    contract; for legal information regarding products (including the MetaBrainz's Databases), please consult the license pages of 
+    the specific project.
+  </p>
+
+  <ol>
+    <li>
+      <b>MetaBrainz's projects will remain 100% free</b>: The development process will be open to the public by using public mailing lists, meetings 
+      open to interested parties, and open IRC channels. The client and server software will remain free and accessible under generally 
+      accepted open licenses. The factual database content will be in the public domain and the non-factual database content will be released 
+      under a Creative Commons license. 
+    </li>
+
+    <li>
+      <b>We will give back to the web and free software community</b>: When we add extensions to MetaBrainz projects we will license them as free 
+      software or content. We will make the best system we can, so that our free encyclopedia will be widely distributed and used. We will 
+      feed back bug-fixes, improvements, user requests, etc. using public mailing lists and/or open meetings. 
+    </li>
+
+    <li>
+      <b>We won't hide problems and policies</b>: We will keep all MetaBrainz related discussions open for public view at all times, regardless 
+      of their content. All problems and policies related to MetaBrainz will be visible to all. 
+    </li>
+
+    <li>
+      <b>Our priorities are our users, free content, and free software</b>: We will be guided by the needs of our users, free content and software 
+      community. We will place their interests first in our priorities. We won't object to commercial use of our content, companies can use 
+      the work of our volunteers without any charge, but charging for the content itself is forbidden. We aim to follow the GPL in that 
+      "You may charge a fee for the physical act of transferring a copy, and you may at your option offer warranty protection in exchange 
+      for a fee." 
+    </li>
+
+    <li>
+      <b>MetaBrainz is interested in all types of music/data</b>: MusicBrainz aims to collect information about all types of music. Published/unpublished, 
+      popular/fringe, western/non-western and human/non-human music should all be entered into MusicBrainz. MetaBrainz/MusicBrainz does not discriminate 
+      on sex, religion, race or controversial viewpoints. MusicBrainz does not prefer one type of music over another, which also means that 
+      MetaBrainz does not condemn or condone any of the music catalogued in the database. Editors and voters are expected to tolerate viewpoints 
+      that may differ from their own. 
```

Comment:
Imho, specific references to MusicBrainz should be removed here:
"
MetaBrainz does not discriminate on sex, religion, race or controversial viewpoints.
Users are expected to tolerate viewpoints that may differ from their own.
"